script.js

```diff
@@ -0,0 +1,8 @@
+const btnMobile = document.getElementById('ham-menu');
+
+function toggleMenu() {
+  const nav = document.getElementById('nav');
+  nav.classList.toggle('nav-active');
+}
```

Comment:
- [ ] Great job working on the functionality but I suggest to  add more margin/padding to the li elements tom match the figam as much as possible ✅🤩💯💪🏻 since in the requirements says You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- Your design 

![image](https://user-images.githubusercontent.com/58553711/231025508-c94ff14c-61f7-431d-acb3-2df7ddf70720.png)

- Figma
![image](https://user-images.githubusercontent.com/58553711/231025551-9ddde92a-c39d-4048-94ad-156fb91322d6.png)

Reply:
Correction made